cmd/tellor/main.go

```diff
@@ -193,16 +193,22 @@ func moveCmd(f func(context.Context, log.Logger, common.Address, *big.Int) error
 }
 
 func balanceCmd(cmd *cli.Cmd) {
+
 	addr := ETHAddress{}
 	cmd.VarArg("ADDRESS", &addr, "ethereum public address")
 	cmd.Spec = "[ADDRESS]"
 	cmd.Action = func() {
+		// Using values from context, until we have a function that setups the client and returns as values, not as part of the context
+		getter := ctx.Value(tellorCommon.ContractsGetterContextKey).(*getter.TellorGetters)
+		client := ctx.Value(tellorCommon.ClientContextKey).(rpc.ETHClient)
+		commonAddress := ctx.Value(tellorCommon.PublicAddress).(common.Address)
```

Comment:
Due to the order the functions were called, the ctx is emprty until cmd.Action() is callled, so this is the place where it needs to be to work as is. I think that's not very good though, as we need a similar structure for all commands. It need to be refactored to account for that. Maybe include that when changing de cli paackage?

Reply:
I am ok with that for the sake to continue the refactoring.

If you confirm that this works when testing locally I am ok to merge it.

Reply:
It's working locally!